lib/ansible/plugins/inventory/docker_machine.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Ximon Eighteen <ximon.eighteen@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: docker_machine
+    plugin_type: inventory
+    short_description: Docker machine inventory source
+    requirements:
+        - Docker Machine
+    extends_documentation_fragment:
+        - inventory_cache
+        - constructed
+    description:
+        - Get inventory hosts from Docker Machine.
+        - Uses a YAML configuration file that ends with docker_machine.(yml|yaml).
+        - The plugin returns an 'all' group of nodes and one group per driver (e.g. digitalocean).
+        - The plugin sets standard host variables ansible_host, ansible_port, ansible_user and ansible_ssh_private_key.
+        - THe plugin also sets standard host variable ansible_ssh_common_args to '-o StrictHostKeyChecking=no'.
```

Comment:
```suggestion
        - The plugin also sets standard host variable ansible_ssh_common_args to '-o StrictHostKeyChecking=no'.
```
Same in the line below.